File: URI1048.py
salario = float(input())
nSalario = 0
rGanho = 0
rPercentual = 0
if 0 < salario <= 400:
    rGanho = salario*0.15
    nSalario = (salario+rGanho)
    rPercentual = 15
elif 400.01 <= salario <= 800:
    rGanho = salario * 0.12
    nSalario = (salario + rGanho)
    rPercentual = 12
elif 800.01 <= salario <= 1200:
    rGanho = salario * 0.10
    nSalario = (salario + rGanho)
    rPercentual = 10
elif 1200.01 <= salario <= 2000:
    rGanho = salario * 0.07
    nSalario = (salario + rGanho)
    rPercentual = 7
else:
    rGanho = salario * 0.04
    nSalario = (salario + rGanho)
    rPercentual = 4

print("Novo salario: {:.2f}".format(nSalario))
print("Reajuste ganho: {:.2f}".format(rGanho))
print("Em percentual: {} %".format(rPercentual))

File: C/URI1048.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main() {
    double salario, reajuste;
    int pct;
    scanf("%lf", &salario);

    if (salario <= 400) {
        reajuste = salario * 0.15;
        salario += reajuste;
        pct = 15;
    } else if ( salario <= 800) {
        reajuste = salario * 0.12;
        salario += reajuste;
        pct = 12;
    } else if (salario <= 1200) {
        reajuste = salario * 0.10;
        salario += reajuste;
        pct = 10;
    } else if (salario <= 2000) {
        reajuste = salario * 0.07;
        salario += reajuste;
        pct = 7;
    } else {
        reajuste = salario * 0.04;
        salario += reajuste;
        pct = 4;
    }

    printf("Novo salario: %.2lf\n", salario);
    printf("Reajuste ganho: %.2lf\n", reajuste);
    printf("Em percentual: %d %\n", pct);



    return 0;
}